ddl/db_change_test.go

```diff
@@ -710,16 +712,33 @@ func (s *testStateChangeSuite) testControlParallelExecSQL(c *C, sql1, sql2 strin
 	c.Assert(err, IsNil)
 	wg.Add(2)
 	ch := make(chan struct{})
+	// Make sure the sql1 is put into the DDLJobQueue.
+	go func() {
+		var qLen int
+		for {
+			kv.RunInNewTxn(s.store, false, func(txn kv.Transaction) error {
+				jobs, err3 := admin.GetDDLJobs(txn)
+				if err3 != nil {
+					return err3
+				}
+				qLen = len(jobs)
+				return nil
+			})
+			if qLen == 1 {
+				// Make sure sql2 is executed after the sql1.
+				close(ch)
+				break
+			}
+			time.Sleep(5 * time.Millisecond)
```

Comment:
Here can not fully ensure that sql2 is executed after sql1, there will be cases that do not meet expectations, so you can adjust it?

Reply:
Why can't do it?
The `close(ch)` only be called in line729. So the sql2 must be done after the sql1 is put in DDLJobQueue.